llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1120,6 +1122,12 @@ def InsnS : DirectiveInsnS<(outs), (ins uimm7_opcode:$opcode, uimm3:$funct3,
                                         AnyReg:$rs2, AnyReg:$rs1,
                                         simm12:$imm12),
                            "$opcode, $funct3, $rs2, ${imm12}(${rs1})">;
+def Insn32 : RVInst<(outs), (ins uimm32:$value), "", "", [], InstFormatOther> {
```

Comment:
Does `InstFormat` matter here?

Reply:
I don't think it should. Only a couple of the possible values are used. I think they are used to pick relocation type in the encoder. That shouldn't matter for these instructions.